eris.go

```diff
@@ -435,6 +435,14 @@ func (e *rootError) Format(s fmt.State, verb rune) {
 
 // Is returns true if both errors have the same message and code. Ignores additional KV pairs.
 func (e *rootError) Is(target error) bool {
```

Comment:
nit: Let's make it clearer to the user.

```suggestion
// Is returns true if both errors have the same message and code.
// In case of a joined error, returns true if at least one of the joined errors is equal to target.
// Ignores additional KV pairs.
func (e *rootError) Is(target error) bool {
```